runbooks/how-to-clone-rds-cluster-with-terraform.md

```diff
@@ -0,0 +1,80 @@
+# How to Clone an RDS Cluster with Terraform
+
+**NOTE**: This runbook **DOES NOT** explicitly cover creating production clones for use as replacements to primary environment data stores.
+
+Follow this runbook to create a clone of an RDS cluster for development purposes.
+This runbook assumes you have sufficient access required to manipulate the remote terraform state stored in AWS S3, the state lock stored in AWS DynamoDB, and general read and write privileges to access the various AWS resources at issue, e.g. AWS SSM Parameter Store, AWS IAM, and AWS RDS among others.
+
+## Software Requirements
+- [tfenv](https://github.com/tfutils/tfenv#installation)
+- [awscli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions)
+
+## Configuring an Ephemeral Environment
+All steps below assume you have the latest `master` in a local working copy of the BFD repository
+
+### Initial Setup
+Define the ephemeral environment (`ephemeralEnv`) and seed environment (`TF_VAR_ephemeral_seed_environment`) environment variables. Optionally, define `TF_VAR_ephemeral_rds_snapshot_id_override`.
+- the `ephemeralEnv`
+    - should be meaningful, easily traceable
+    - is used as the workspace name, as `$env` in e.g. resource identifiers `bfd-${env}-fhir`
+    - could be the numeric portion of the JIRA key associated with your work, e.g. BFD-2090 becomes `2090`
+- `TF_VAR_ephemeral_seed_environment` seed environment is the targeted, _existing_ environment you intend to copy, e.g. `prod-sbx`
```

Comment:
> Should this be `ephemeral_environment_seed`?

Fine question! Here, I'm providing a preview of the **Example** just below. In this context, I've elected to limit initial setup to manipulating environment variables to inform the workspace name via `$ephemeralEnv` as well as providing input variables the similarly with the [`TF_VAR_` prefix](https://developer.hashicorp.com/terraform/cli/config/environment-variables#tf_var_name).

Do you think that...
- this should be more explicit?
- this could be clearer by e.g. transforming `terraform apply` to `terraform apply -var=ephemeral_environment_seed=2090` instead?

Reply:
Ah, no, I just meant that there's a variable is defined as `ephemeral_environment_seed` in `ops/terraform/services/base/variables.tf` and not `ephemeral_seed_environment` you've used in the markdown file. Maybe I'm misunderstanding something and they're not related?

Reply:
> [...] there's a variable defined as `ephemeral_environment_seed` in `ops/terraform/services/base/variables.tf` and not `ephemeral_seed_environment` you've used in the markdown file.

Let me fix that 🤦

> Maybe I'm misunderstanding something and they're not related?

Nope. You're on the money. Thanks!